tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -60,6 +60,14 @@ afterAll(async () => {
 });
 
 describe(`AI Lab extension installation and verification`, async () => {
+  describe(`Verify Podman Machine details`, async () => {
```

Comment:
Add this check after extension installation.

Reply:
Or add a check for podman machine is present the same way as here: https://github.com/containers/podman-desktop-extension-bootc/blob/main/tests/playwright/src/bootc-extension.spec.ts#L68